kernel/Kconfig.power_mgmt

```diff
@@ -49,7 +49,7 @@ config SYS_POWER_LOW_POWER_STATE
 	  saving most power.
 
 config SYS_POWER_DEEP_SLEEP
-	bool "Deep sleep state"
+	bool "Enable Deep Sleep states"
```

Comment:
"Enable" in Kconfig option is implied, i.e. all the options are there to be enabled. There is no need to be explicit or otherwise all the Kconfig options would start with "Enable ...". It's better to have
```
	bool "Deep Sleep states"
```

Reply:
Fixed